__tests__/app.test.js

```diff
@@ -282,4 +283,33 @@ describe("POST /api/articles/:article_id/comments", () => {
         );
       });
   });
+
+  test("400: Responds with a 400 when there is no body", () => {
+    const newComment = {
+      username: "rogersop"
+    };
+    return request(app)
+      .post(`/api/articles/1/comments`)
+      .send(newComment)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("400 - Bad request, you haven't typed a comment!");
+      });
+  });
+
+  test("400: Responds with a 400 when there are more than two key-value pairs", () => {
+    const newComment = {
+      username: "rogersop",
+      body: "AHHHHHHHH test banana grapefruit AHHHH!!!",
+      additionalKey: "additionalValue"
+    };
+    return request(app)
+      .post(`/api/articles/1/comments`)
+      .send(newComment)
+      .expect(400)
+      .then((res) => {
+        expect(res.body.msg).toBe("400 - Bad request, enter a single username and comment");
+      });
+  });
+
```

Comment:
This is nice to see included, but this shouldnt fail with an error of 400. You should actually still give back a 200 with the new comment added but simply ignoring the extra key value pairs.